src/ui/src/builder/BuilderFieldsTools.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+	<div
+		class="BuilderFieldsTools colorTransformer"
+		:data-automation-key="props.fieldKey"
+	>
+		<div class="tools">
+			<div
+				v-for="(tool, toolName) in tools"
+				:key="toolName"
+				class="tool"
+				@click="editTool(toolName)"
+			>
+				<div class="toolName">{{ toolName }}</div>
+				<button class="delete" @click.stop="deleteTool(toolName)">
+					<i class="material-symbols-outlined">delete</i>
+				</button>
+			</div>
+		</div>
+
+		<WdsButton
+			variant="builder"
```

Comment:
This variant doesn't exist, no ?

https://github.com/writer/writer-framework/blob/aa9b61cdfe2a79e4fce7fed88962f9e407598d21/src/ui/src/wds/WdsButton.vue#L11